package-lock.json

```diff
@@ -9481,25 +9555,18 @@
 			}
 		},
 		"vscode": {
-			"version": "1.1.26",
-			"resolved": "https://registry.npmjs.org/vscode/-/vscode-1.1.26.tgz",
-			"integrity": "sha512-z1Nf5J38gjUFbuDCbJHPN6OJ//5EG+e/yHlh6ERxj/U9B2Qc3aiHaFr38/fee/GGnxvRw/XegLMOG+UJwKi/Qg==",
+			"version": "1.1.33",
```

Comment:
Where are these changes coming from?

Reply:
I'm assuming they come from an npm install after I merge from master

Reply:
Either way they shouldn't be a part of a PR where you didn't change "package.json". You either need to merge this branch with master or clean up somehow